src/location.rs

```diff
@@ -311,14 +326,22 @@ pub async fn set(context: &Context, latitude: f64, longitude: f64, accuracy: f64
 
 pub async fn get_range(
     context: &Context,
-    chat_id: ChatId,
-    contact_id: u32,
+    chat_id: Option<ChatId>,
+    contact_id: Option<u32>,
     timestamp_from: i64,
     mut timestamp_to: i64,
 ) -> Vec<Location> {
     if timestamp_to == 0 {
         timestamp_to = time() + 10;
     }
+    let (disable_chat_id, chat_id) = match chat_id {
+        Some(chat_id) => (0, chat_id),
+        None => (1, ChatId::new(0)), // this ChatId is unused
```

Comment:
Shouldn't it be `bool` instead of `1` and `0` for `disable_chat_id` and `disable_contact_id`?

Reply:
sqlite does not have a boolean datatype, https://www.sqlite.org/datatype3.html#boolean_datatype , however, might be that paramsv handles that somehow. but maybe it is fine as it is. not to use specials available in the wrapper but not in sqlute maybe also make conversion to sqlx easier.